### PR TITLE
Changes for FTS Public

### PR DESCRIPTION
**Pull request by Pandya-Hardik**
I have changed the values of the default params as per the documentation. Also, changed the function name from ocha_snap_pdf to ocha_snap to make it sound more generic.

### CODE REVIEW

File: ocha_snap.module [hunk 1]
```diff
@@ -12,13 +12,13 @@
  */
 
 // A default snap service endpoint.
-define('OCHA_SNAP_URL', 'http://localhost:8442/snap');
+define('FTS_DATA_SNAP_URL', 'http://snap:8442/snap');
```
Comment (berliner) [inline ocha_snap.module]: This shouldn't be part of this PR

File: ocha_snap.module [hunk 2]
```diff
@@ -12,13 +12,13 @@
...
 
 /**
  * Helper to fetch the snap service endpoint.
  */
 function ocha_snap_url() {
-  return variable_get('ocha_snap_url', OCHA_SNAP_URL);
+  return variable_get('fts_data_snap_url', FTS_DATA_SNAP_URL);
```
Comment (berliner) [inline ocha_snap.module]: Same

File: ocha_snap.module [hunk 3]
```diff
@@ -41,14 +41,14 @@ function ocha_snap_credentials() {
  * PDF generator.
  *
  * @param string $url
- *   The URL to generate the PDF for.
+ *   The URL to generate the PDF /PNGfor.
```
Comment (berliner) [inline ocha_snap.module]: missing space

**Review by berliner — CHANGES_REQUESTED** [review]

File: ocha_snap.module (hunk 1 above)
Comment (Pandya-Hardik) [reply ocha_snap.module]: Yes, I have reverted this change. Note, using the url (http://localhost:8442/snap) defined in here does not work on my local. Hence, I had changed this based on Peter's comment https://humanitarian.atlassian.net/browse/HPC-5797?focusedCommentId=99367&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-99367

@cafuego can guide us better here.

Comment (berliner) [reply ocha_snap.module]: Just use `OCHA_SNAP_URL`here, and then use `variable_get('ocha_snap_url', OCHA_SNAP_URL )` later in the code. My earlier change request for these names are not valid anymore, as this is not our module and we should respect the naming conventions here and not introduce FTS specific terminology.

File: ocha_snap.module (hunk 2 above)
Comment (berliner) [reply ocha_snap.module]: Same as above.

File: ocha_snap.module (hunk 3 above)
Comment (Pandya-Hardik) [reply ocha_snap.module]: I have added/reverted the changes as suggested.

File: ocha_snap.module [hunk 4]
```diff
@@ -41,14 +41,14 @@ function ocha_snap_credentials() {
  * PDF generator.
  *
  * @param string $url
- *   The URL to generate the PDF for.
+ *   The URL to generate the PDF /PNG for.
```
Comment (cafuego) [inline ocha_snap.module:44]:
```suggestion
 *   The URL to generate the PDF or PNG from.
```

File: ocha_snap.module [hunk 5]
```diff
@@ -41,14 +41,14 @@ function ocha_snap_credentials() {
...
  * @param array $params
  *   An array of optional parameters for the Snap Service request.
  *
  * @return various
  *   A binary PDF data blob or FALSE if an error occurred.
  */
-function ocha_snap_pdf($url, $params = array()) {
+function ocha_snap($url, $params = array()) {
```
Comment (cafuego) [inline ocha_snap.module:51]: Yep, fair :-)

**Review by cafuego — APPROVED** [review]
With the amendments this looks OK. Just a minor tweak to the doc change for the main function, which I've done. Thanks @Pandya-Hardik

File: ocha_snap.module (hunk 1 above)
Comment (Pandya-Hardik) [reply ocha_snap.module]: @cafuego Using the value of `OCHA_SNAP_URL` as `'http://localhost:8442/snap'` does not work as based on your comment https://humanitarian.atlassian.net/browse/HPC-5797?focusedCommentId=99367&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-99367 we have to use `'http://snap:8442/snap'`

I suppose this has to be changed.

Comment (Pandya-Hardik) [reply ocha_snap.module]: @cafuego Regarding the change of value for OCHA_SNAP_URL, we could handle this via settings.local.inc or update hook at our end and nothing would be needed to be amended. Thanks for the module and your support!
